### ISSUE TITLE
bug (hubble): "No Data"

### ISSUE THREAD

**Issue by liiiiiiii99** [farcasterxyz/hub-monorepo#2094]
**What is the bug?**
I installed it using this command ```curl -sSL https://download.thehubble.xyz/bootstrap.sh | bash``` but it is showing "No Data".

**How can it be reproduced? (optional)**
N/A

**Additional context (optional)**
For context, I'm using WSL and Docker Desktop. and on Docker, here's the last line ```2024-06-25 19:15:03 {"message":"Please wait... This may take several minutes\n","timestamp":"2024-06-25T11:15:03.502Z","type":"out","process_id":0,"app_name":"hubble"}``` and it seems to just be repeating it after a few minutes.

![image](https://github.com/farcasterxyz/hub-monorepo/assets/169754563/bb69f31e-02d6-4d6b-a8b1-8958d1ae61af)

**Comment by totoyc1** [comment]
And i think it restart automatically for some reason after showing the
```2024-06-25 19:15:03 {"message":"Please wait... This may take several minutes\n","timestamp":"2024-06-25T11:15:03.502Z","type":"out","process_id":0,"app_name":"hubble"}```

a few minutes it shows 
```2024-06-25 19:16:21 {"message":"{\"level\":30,\"time\":1719314181662,\"pid\":29,\"hostname\":\"965e429d6e26\",\"msg\":\"Hubble stopped, exiting normally\"}\n","timestamp":"2024-06-25T11:16:21.662Z","type":"out","process_id":0,"app_name":"hubble"}
2024-06-25 19:16:21 2024-06-25T11:16:21: PM2 log: App name:hubble id:0 disconnected
2024-06-25 19:16:21 2024-06-25T11:16:21: PM2 log: App [hubble:0] exited with code [1] via signal [SIGINT]
2024-06-25 19:16:21 2024-06-25T11:16:21: PM2 log: 0 application online, retry = 3
2024-06-25 19:16:23 2024-06-25T11:16:23: PM2 log: 0 application online, retry = 2
2024-06-25 19:16:25 2024-06-25T11:16:25: PM2 log: 0 application online, retry = 1
2024-06-25 19:16:27 2024-06-25T11:16:27: PM2 log: 0 application online, retry = 0
2024-06-25 19:16:27 2024-06-25T11:16:27: PM2 log: Stopping app:hubble id:0```

and just repeat in a loop.  for some reason

**Comment by youngatsavings1** [comment]
same issue to me

**Comment by youngatsavings1** [comment]
hubble-1  | 2024-06-25T21:07:01: PM2 log: App name:hubble id:0 disconnected
hubble-1  | 2024-06-25T21:07:01: PM2 log: App [hubble:0] exited with code [1] via signal [SIGINT]
hubble-1  | 2024-06-25T21:07:03: PM2 log: 0 application online, retry = 3
hubble-1  | 2024-06-25T21:07:05: PM2 log: 0 application online, retry = 2
hubble-1  | 2024-06-25T21:07:07: PM2 log: 0 application online, retry = 1
hubble-1  | 2024-06-25T21:07:09: PM2 log: 0 application online, retry = 0
hubble-1  | 2024-06-25T21:07:09: PM2 log: Stopping app:hubble id:0
hubble-1  | 2024-06-25T21:07:09: PM2 log: pid=29 msg=process killed
hubble-1  | 2024-06-25T21:07:09: PM2 log: PM2 successfully stopped
hubble-1 exited with code 0

**Comment by BlunderChia** [comment]
Same issue here, has anyone found a solution to this?

**Comment by totoyc1** [comment]
Not yet sadly. :(

**Comment by BlunderChia** [comment]
Probably it is because of pm2 log rotation. Dev team should build a workaround to the installation script.

**Comment by kingo233** [comment]
Same issue, anyone can help me?

**Comment by ertan2002** [comment]
a very professional team is on the way haha.. sorry but this is very silly. I've tired with many computers but got the same error always and always

**Comment by prasadchinthala01** [comment]
facing same issu plz help
![hubble 1](https://github.com/farcasterxyz/hub-monorepo/assets/174694388/f1dad84a-9a6a-496d-9172-fc7f2dc6ca6e)

**Comment by youngatsavings1** [comment]
Bro i found solution just reinstall your vps and make sure you are using ubuntu 24.04 and let me know if it works. All the best

**Comment by sds** [comment]
Hey everyone, we are aware of an issue where if your internet connection is interrupted at some point during the process, the snapshot can become corrupted and then the hubs end up in a restart loop. 

If you see something like:
>db.internal_error/Corruption: Corruption: IO error: No such file or directory: While open a file for random read: .rocks/rocks.hub._default/004134.sst: No such file or directory  The file .rocks/rocks.hub._default/MANIFEST-000005 may be corrupted.

...in your logs, this is likely what's affecting you.

We're tracking this issue in #2042.

**Comment by sanjayprabhu** [comment]
This should be fixed by https://github.com/farcasterxyz/hub-monorepo/pull/2236, included in the latest release (1.14.3).
